.slack/slack.json

```
@@ -8,5 +8,14 @@
     "script": {
       "default": "deno run -q --unstable --import-map=import_map.json --allow-read --allow-write --allow-net https://deno.land/x/deno_slack_builder@0.0.5/mod.ts"
     }
+  },
+  "run": {
+    "script": {
+      "default": "deno run -q --unstable --allow-write --allow-read --allow-net https://deno.land/x/deno_slack_runtime@0.0.2/mod.ts"
+    },
+    "watcher": {
+      "filter_regex": "^.*\\.ts$",
+      "paths": [ "./manifest.ts" ]
```

Do we want to support all supported types like the builder?
```suggestion
      "paths": [ "./manifest.json", "./manifest.js", "./manifest.ts" ]
```

---

➕ ➕ ➕  but I'm not sure if this also requires an update to the `filter_regex`

I think this should be the standard provided via `get-hooks` as well

---

For deno / run-on-slack projects, we can just drop the regex altogether (based on how the [slack-cli handles this property](https://github.com/slackapi/slack-cli/blob/main/pkg/cloud/run.go#L242-L245) - it is ignored if it is missing) since we can, as @mwbrooks suggests, just explicitly reference all three possible manifest filetypes. In these projects, since each function-execution event coming from the Slack backend causes the CLI to spin up a new `deno` process wrapping the user function execution (the short-lived process model, as opposed to Bolt, which uses a long-lived process model), we only need to worry about changes to the manifest files here.